plugins/parsers/prometheus/parser.go

```diff
@@ -186,3 +187,19 @@ func (p *Parser) GetTimestamp(m *dto.Metric, now time.Time) time.Time {
 	}
 	return t
 }
+
+func (p *Parser) Init() error {
+	return nil
+}
+
+func (p *Parser) InitFromConfig(config *parsers.Config) error {
+	p.IgnoreTimestamp = config.PrometheusIgnoreTimestamp
+	return p.Init()
+}
```

Comment:
You can kill `Init` here and just return `nil`

```suggestion
func (p *Parser) InitFromConfig(config *parsers.Config) error {
	p.IgnoreTimestamp = config.PrometheusIgnoreTimestamp
	return nil
}
```